### ISSUE TITLE
Support custom styling for line plots

### ISSUE THREAD

**Issue by mtcliatt** [mtcliatt/flarts#5]

**Comment by JayPerfetto** [comment]
+1 -> is there any way to apply coloring to the background of the chart or change axis label text color?

**Comment by thedejifab** [comment]
@mtcliatt, anything yet with regards to @JayPerfetto's question?

**Comment by mtcliatt** [comment]
Sorry for the late reply, that functionality doesn't exist yet but wouldn't be difficult to add.
There's another charting package which does have that functionality (I'm pretty sure), which may suit your needs better: https://github.com/imaNNeoFighT/fl_chart

I haven't added this yet because I've been working on scrolling, panning, and zooming, which turns out to be a really hard problem with labeling and keeping the view zoomed to fit. I'm almost done, but because that other package looks like it works so nicely, I may turn the focus of this package to charting for stocks and related data.

Let me know if FL_Chart works for y'all.

**Comment by thedejifab** [comment]
Yeah. I've tried using Fl_Chart but it doesn't really give me the stocks feel. Another thing is having a segment under the line chart which can have a gradient colour with opacity  @mtcliatt. How quick are we going to get a modification in that regard, because I'm currently working on a project that needs stuff around that?

**Comment by mtcliatt** [comment]
I can probably do that this week.
